### DISCUSSION TITLE
Copy column data

### DISCUSSION THREAD

**Discussion by paur94** [KevinVandy/material-react-table#536] [Q&A]
Hi there,
Can I click or select and copy ALL column data? Not one cell, all column data.
Thanks

**Comment by joey-kwl** [comment]
No I don't think it's possible. What you can do is create your own button that uses the `navigator.clipboard.writeText` [web api](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText). 

And with the [Row selection state](https://www.material-react-table.com/docs/guides/row-selection#access-row-selection-state) you know which row is selected

**Comment by KevinVandy** [comment]
Just map through your data and write to clipboard in the format you want.

**Comment by paur94** [comment]
For example, I have this column
![image](https://github.com/KevinVandy/material-react-table/assets/60259252/08f8f269-e4b5-4af9-8f3c-f53654852cba)

and I want a header on click copy that column data, how can I map through that column data?
